iroha/src/bridge.rs

```diff
@@ -1 +1,93 @@
 //! This module contains functionality related to `Bridge`.
+
+use crate::prelude::*;
+use iroha_derive::Io;
+use parity_scale_codec::{Decode, Encode};
+use std::fmt::Debug;
+
+/// Enumeration of all supported bridge kinds (types). Each variant represents some
+/// distributed ecosystem or protocol which can be used within Iroha.
+#[derive(Encode, Decode, PartialEq, Eq, Debug, Clone, Copy, Hash, Io)]
+pub enum BridgeKind {
+    /// [Polkadot](https://polkadot.network/) network.
+    Polkadot,
```

Comment:
It is not a kind of bridge. Bridge kind here is a used protocol. For we can call what we trying to implement XClaim_transfer because it implements transfer flow of XClaim. Polkadot and more precisely relay Chain it will be the title of this bridge when it will be instantiated in runtime. As we discussed it should be possible to use this protocol for multiple bridges, But it always will be current realization....

Reply:
@yurivin right. Renamed to IClaim (by analogy with XClaim).